.github/workflows/ci.yml

```diff
@@ -48,20 +48,24 @@ jobs:
     needs: ["create-semantic-tag"]
     uses: "./.github/workflows/test.yml"
 
+  performance:
```

Comment:
Why put it in a separate workflow? It's part of the tests

Reply:
I can make this workflow part of the test again. 

in my approach, the job "build" started after the necessary unit tests and did not wait for the performance tests to be completed

Reply:
Performance tests are not that long